docs-ref-conceptual/release-notes-azure-cli.md

```diff
@@ -12,6 +12,60 @@ ms.devlang: azurecli
 ---
 
 # Azure CLI release notes
+## April 23, 2019
+
+Version 2.0.63
+
+### ACS
+* "az aks get-credentials" will prompt to overwrite duplicated values
+* Remove "(PREVIEW)" from Dev Spaces commands, "az aks use-dev-spaces" and "az aks remove-dev-spaces".
+
+### AMS
+* Bug fix for asset and account filters update.
+
+### AppService
+* webapp: az webapp ssh now support ASE and a timeout parameter
+* functionapp: add feature for establishing CI CD to an Azure DevOps pipeline from a Github repository
+* functionapp: in `az functionapp devops-build create`, added `--github-pat` flag to accept Github personal access token
+* functionapp: in `az functionapp devops-build create`, added `--github-repository` flag to accept Github repository that contains a functionapp source code
+* webapp: az webapp up --logs was failing with a error and updating default .NETCORE version to 2.1
+* functionapp: remove unnecessary app settings while creating a function app with consumption plan
+* webapp: az webapp up default asp string now appends number at the end to create a new ASP based on SKU options
+* webapp: az webapp up support -b as an option to launch the app in the browser
+* webapp: az webapp deployment source config zip handles 'AZURE_CLI_DISABLE_CONNECTION_VERIFICATION' environment variable
+
+### Deployment Manager
+* Preview release. Create and manage artifacts that support rollouts.
+
+### Lab
+* Fixed error: 'ObjectsOperations' object has no attribute 'get_current_user'
+
+### Network
+* `dns zone create`: Add auto name server delegation in parent during child zone creation
+
+### Resource
+* `resource link`: Deprecate `--link-id`, `--target-id` and `--filter-string` in favor of
+                   `--link`, `--target`, and `--filter` respectively.
+* `resource link create/update`: Fix issue where these commands would not work.
+* `resource delete`: Fix issue where deleting using a resource ID could crash on error.
+
+### Role
+* [BREAKING CHANGE] 'group create'/'app create'/'sp create-for-rbac': supports idempotent, hence confliction error will not be surfaced
```

Comment:
Unintelligible; sorry, I need to actually know what this change does and what it means for users. __Release notes cannot be merged until this is resolved.__

Reply:
The breaking change is from PR: https://github.com/Azure/azure-cli/pull/9108.
It fixes 3 issues:
https://github.com/Azure/azure-cli/issues/8750
https://github.com/Azure/azure-cli/issues/8519
https://github.com/Azure/azure-cli/issues/9032

Does it make clear? If not, @yugangw-msft can explain more details as he made the changes.

Reply:
Is this truly a breaking change? The issues linked make it look like a bugfix of a feature that was misbehaving. cc @yugangw-msft

Reply:
This is a breaking change, for 90% users it is a good break though. I hope the `breaking` wording would reminder them that they no longer need the `try-catch` blocks any more.

Reply:
End users do not have to use try-catch blocks. This sounds like it was an engineering-focused change. Please don't include those in the release notes.

Removing.